db/db-challenge3/db-013/013.sql

```diff
@@ -1,6 +1,6 @@
 BEGIN;
 UPDATE users
 SET is_deleted = 1
-WHERE mobile_tel is NULL
-AND business_tel is NULL;
-COMMIT;
\ No newline at end of file
+WHERE mobile_tel = ' '
```

Comment:
```suggestion
WHERE mobile_tel = ' '
WHERE mobile_tel = ''
```

もともとの書き方ですと、「半角スペース1文字」にマッチする、という条件になっています。
また、この指摘の通りに修正しても、長田さんからの指摘事項は反映できていません。

> というのは、「NULLか空文字のいずれかの場合」と取ってもらえると。

調べてみてください。（わからなかったら質問に来てもらえたら。）